tools/shared.py

```diff
@@ -2299,6 +2299,12 @@ def reconfigure_cache():
   global Cache
   Cache = cache.Cache(debug=DEBUG_CACHE)
 
+# Temporary workaround for SINGLE_FILE, pending refactoring
+class FilenameReplacementStrings:
+  WASM_TEXT_FILE = 'FILENAME_REPLACEMENT_STRINGS_WASM_TEXT_FILE'
```

Comment:
Got it, thanks, I meant to ask about how to handle this since it seemed a bit fragile. (Probably should've picked up on that convention from jsifier.js.)